### ISSUE TITLE
levelup events not firing

### ISSUE THREAD

**Issue by bhoriuchi** [pgte/skiff#41]
Playing around with skiff (using memdown) and I am attempting to get levelup put/del events from a 3 node skiff cluster. i have 1 node making a put request after the leader has been elected and i can see that the data is written. All 3 nodes run the same code on different ports in separate processes with the exception of node 3 calling a put after 2.5s

pertinent code is

```js
const db = skiff.levelup()
db.on('put', console.log)
```

I get no console output on any of the 3 nodes. Is there a way to achieve this?

**Comment by pgte** [comment]
Yes, this looks like a bug. There needs to be a notification of changes bubbling up from the internal db into this interface.
